lib/ansible/plugins/doc_fragments/url_redirect.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would personally not add a new docs fragment, but add a sub-docs fragment to the `url` docs fragment. Not sure what the Core team prefers though.

Reply:
I cannot find any docs on this, but it sounds like the way to go. Could you give me a pointer on how to implement it?

Reply:
See for example:
- https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/doc_fragments/action_common_attributes.py#L53
- https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/import_playbook.py#L26

Reply:
I added the sub-doc fragment. I'm unsure about the uri-lookup docs though. The default there is `urllib2` while the uri-module has `safe` as default. Would this be overwritten in a wrong manner?